src/lib.rs

```diff
@@ -1227,4 +1275,28 @@ mod tests {
         let b = Bdf::new(Bus(255), Dev(31), Func(7));
         assert_eq!(b.next(), None);
     }
+
+    #[test]
+    fn test_log() {
+        let _ = env_logger::builder()
+            .is_test(true)
+            .filter_level(LevelFilter::Info)
+            .try_init();
+
+        let device: Device = Device {
+            ops: PciEcamCfgOps {
+                segment_id: 0xA0FF,
+                base: 0x0000,
+            },
+            bdf: Bdf::new(Bus(0), Dev(0), Func(2)),
+            resources: [None; 6],
+            header: HeaderType::Normal,
+            vendor_id: 0x4,
+            device_id: 0xc,
+            device_info: DeviceInfo::new(1, 8, 2),
+            revision: 2,
+        };
+
+        device.print_pci_config_space();
+    }
```

Comment:
Does this test run successfully on your host machine?

Reply:
I realized it fails due to the comment, I didn't know it runs lines in doc for testing. fixing it